rest/kadai-rest-spring/src/main/java/io/kadai/common/rest/ldap/LdapClient.java

```diff
@@ -384,11 +384,11 @@ public List<AccessIdRepresentationModel> searchPermissionsAccessIdHas(final Stri
     }
     orFilter.or(new EqualsFilter(getPermissionsOfUserName(), dn));
     final AndFilter andFilter2 = new AndFilter();
-    andFilter2.and(new PresentFilter(getUserPermissionsAttribute()));
+    andFilter2.and(new PresentFilter(getPermissionNameAttribute()));
     andFilter.and(orFilter);
     andFilter2.and(andFilter);
 
-    String[] userAttributesToReturn = {getUserIdAttribute(), getUserPermissionsAttribute()};
+    String[] userAttributesToReturn = {getUserIdAttribute(), getPermissionNameAttribute()};
```

Comment:
here

Reply:
We checked it and changed the variable name to make more sense